fem/fespace.hpp

```diff
@@ -587,6 +587,35 @@ class FiniteElementSpace
    bool Conforming() const { return mesh->Conforming() && cP == NULL; }
    bool Nonconforming() const { return mesh->Nonconforming() || cP != NULL; }
 
+   /** Set the prolongation operator of the space to an arbitrary sparse matrix,
+       creating a copy of the argument. */
+   void SetProlongation(const SparseMatrix* p)
```

Comment:
To keep the header file more readable, functions with multiple lines usually go in the cpp file.

Reply:
I moved the implementations to the source file in the latest version.